interface.html

```diff
@@ -527,9 +530,21 @@ <h4 class="panel-title collapsed" role="button" data-toggle="collapse" data-pare
                   <div class="col-sm-4 control-label">
                     <label for="fl-store-releaseNotes">Release Notes</label>
                     <p class="help-block label-helper"><small>When submitting an update to your app, use the Release Notes to tell users what is new in the new version.</small></p>
+                    <p class="help-block label-helper">
+                    <a href="#" data-template="fl-store-releaseNotes">Use template</a>
```

Comment:
Can we have some consistency between the two template links?

This one doesn't need its own help-block and can go inside the `<small>` of the previous block with a `<br>`.